lv_conf_template.h

```diff
@@ -905,6 +905,23 @@
 /*==================
  * DEVICES
  *==================*/
+#  define LV_USE_WAYLAND       0
+
+#if LV_USE_WAYLAND
+
+#define LV_DISP_DEF_REFR_PERIOD  LV_DEF_REFR_PERIOD
```

Comment:
Can it be removed?

Reply:
This PR was derived from lv_drivers/wayland. @etag4048 @daniel-rossier, please review this PR or open a new PR for LVGL9 if you have prepared. thank you.

Reply:
> Can it be removed?

can be removed.